Owncloud iOs Client/Owncloud iOs Client-Prefix.pch

```diff
@@ -29,6 +29,6 @@
 #ifdef DEBUG_MODE
 #define DLog( s, ... ) NSLog( @"<%p %@:(%d)> %@", self, [[NSString stringWithUTF8String:__FILE__] lastPathComponent], __LINE__, [NSString stringWithFormat:(s), ##__VA_ARGS__] )
 #else
-#define DLog( s, ... )
+#define DLog( s, ... ) NSLog( @"<%p %@:(%d)> %@", self, [[NSString stringWithUTF8String:__FILE__] lastPathComponent], __LINE__, [NSString stringWithFormat:(s), ##__VA_ARGS__] )
```

Comment:
Why did you changed the way logs are printed in non-debug mode?

Reply:
I forgot remove it. Was to get all logs also on release mode